src/main/java/com/checkout/payments/apm/KlarnaSource.java

```diff
@@ -0,0 +1,85 @@
+package com.checkout.payments.apm;
+
+import com.checkout.apm.klarna.KlarnaProduct;
+import com.checkout.common.CountryCode;
+import com.checkout.common.PaymentSource;
+import com.checkout.payments.RequestSource;
+import com.google.gson.annotations.SerializedName;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder
+public final class KlarnaSource implements RequestSource {
+
+    private static final PaymentSource type = PaymentSource.KLARNA;
+
+    @SerializedName("authorization_token")
+    private final String authorizationToken;
+
+    private final String locale;
+
+    @SerializedName("purchase_country")
+    private final CountryCode purchaseCountry;
+
+    @SerializedName("tax_amount")
+    private final Integer taxAmount;
```

Comment:
Please change to Long and make the attributes non-final (just for consistency). Please do the same on all `RequestSource` classes (even the ones already implemented).

Immutability is technically important, but making these attributes non-final will probably be a more flexible approach for the users of the SDK.